docs/src/architectures/nanopet.rst

```diff
@@ -0,0 +1,110 @@
+.. _architecture-nanopet:
+
+NanoPET
+=======
+
+.. warning::
+
+  This is an **experimental model**.  You should not use it for anything important.
+
+This is a more reader- and user-friendly implementation of PET, with slightly
+improved training and evaluation speed.
```

Comment:
```suggestion
NanoPET
=======

.. warning::

  This is an **experimental model**.  You should not use it for anything important.

This is a re-implementation of the original PET (which lives in https://github.com/spozdn/pet), with slightly improved training and evaluation speed.
```

Reply:
I think the reader-friendlyness is the most important bit actually. If one wants to look into PET's code to understand the architecture, nanoPET is the place to go

Reply:
maybe, but metatrain users don't care about that =D